src/groundlight/cli.py

```diff
@@ -0,0 +1,44 @@
+from functools import wraps
+from typing import Union
+
+import typer
+from typing_extensions import get_origin
+
+from groundlight import Groundlight
+
+cli_app = typer.Typer()
```

Comment:
This change should make it so that `groundlight` or `groundlight -h` or `groundlight --help` all give the help message.

```suggestion
cli_app = typer.Typer(no_args_is_help=True, context_settings={"help_option_names": ["-h", "--help"]})
```